packages/actions/src/eth/ethSendRawTransaction.spec.ts

```
@@ -0,0 +1,146 @@
+    await setAccountHandler(client)({
+      address: impersonatedAddress.toString(),
+      balance: parseEther('1000'),
+    })
+    await client.setImpersonatedAccount(impersonatedAddress.toString())
+
+    const tx = TransactionFactory.fromTxData({
+      nonce: '0x00',
+      maxFeePerGas: '0x09184e72a000',
+      maxPriorityFeePerGas: '0x09184e72a000',
+      gasLimit: '0x2710',
+      to: createAddress('0x' + '43'.repeat(20)),
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)